openapi/spec/openapi.json

```
@@ -2593,6 +2626,7 @@
           },
           "errors": {
             "type": "array",
+            "description": "A list of errors (deprecated - please use failures instead)",
```

Tho it's good to have it in the description, the spec also supports a `deprecated: true` prop. I think it's worth adding that. That also enables us to update codegens to add the notice to docs and tooling.

```suggestion
            "description": "A list of errors (deprecated - please use failures instead)",
            "deprecated": true,
```